Flaky test: TestRunAddHost

```
=== CONT  TestRunAddHost
    run_linux_test.go:108: assertion failed: error is not nil: host was not added, found 0: # <nerdctl>
        127.0.0.1	localhost localhost.localdomain
        :1		localhost localhost.localdomain
        10.0.0.1        test1 test
        10.4.0.140      07fc66977787 alpine-07fc6
        10.4.0.141      ca0911dbb51c alpine-ca091
        127.0.0.1       localhost localhost.localdomain
        :1              localhost localhost.localdomain
        # </nerdctl>
--- FAIL: TestRunAddHost (0.96s)
```

--- a/cmd/nerdctl/run_linux_test.go
+++ b/cmd/nerdctl/run_linux_test.go
@@ -87,7 +87,7 @@ func TestRunIpcHost(t *testing.T) {
 }
 
 func TestRunAddHost(t *testing.T) {
-	t.Parallel()
+	// Not parallelizable (https://github.com/containerd/nerdctl/issues/1127)
 	base := testutil.NewBase(t)
 	base.Cmd("run", "--rm", "--add-host", "testing.example.com:10.0.0.1", testutil.AlpineImage, "cat", "/etc/hosts").AssertOutWithFunc(func(stdout string) error {
 		var found bool

